pkg/cli/testdata/zip/unavailable

```diff
@@ -14,9 +14,15 @@ requesting data for debug/liveness... writing: debug/liveness.json
 requesting data for debug/settings... writing: debug/settings.json
 requesting data for debug/reports/problemranges... writing: debug/reports/problemranges.json
 retrieving SQL data for crdb_internal.cluster_queries... writing: debug/crdb_internal.cluster_queries.txt
+writing: debug/crdb_internal.cluster_queries.txt.err.txt
+  ^- resulted in ...
 retrieving SQL data for crdb_internal.cluster_sessions... writing: debug/crdb_internal.cluster_sessions.txt
+writing: debug/crdb_internal.cluster_sessions.txt.err.txt
+  ^- resulted in ...
 retrieving SQL data for crdb_internal.cluster_settings... writing: debug/crdb_internal.cluster_settings.txt
 retrieving SQL data for crdb_internal.cluster_transactions... writing: debug/crdb_internal.cluster_transactions.txt
+writing: debug/crdb_internal.cluster_transactions.txt.err.txt
```

Comment:
cc @knz in case this is something that needs fixing (see @jordanlewis's explanation on the main page)